spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -64,6 +64,6 @@ public Resource resolve(String location, ResourceLoader resourceLoader) {
 		if (location.startsWith("gs://")) {
 			return new GoogleStorageResource(this.storage, location);
 		}
-		return this.delegate.getResource(location);
+		return null;
```

Comment:
I can revert that and send another PR, but it was breaking a test. The reason for is that when I wrote the first time I really did not check that I could actually return `null` to prevent this protocol resolver from trying to load a resource. The old code, actually worked by luck to be honest, returning the delegate could actually result in StackOverflow, somehow on the old code the only resource loaded was `gs://` so it would never delegate, but on the new test scenario when any other resource was loaded first, it would cause a recursion on the delegate call to getResource.

I should have not changed, as it was not the intention of this PR, but I thought it would not hurt. 

I can revert back, but as I said test will break